paasta_tools/cli/cmds/rollback.py

```diff
@@ -19,8 +22,40 @@
 from paasta_tools.cli.utils import lazy_choices_completer
 from paasta_tools.cli.utils import list_services
 from paasta_tools.generate_deployments_for_service import get_instance_config_for_service
+from paasta_tools.remote_git import list_remote_refs
+from paasta_tools.utils import datetime_from_utc_to_local
+from paasta_tools.utils import format_table
 from paasta_tools.utils import get_git_url
 from paasta_tools.utils import PaastaColors
+from paasta_tools.utils import parse_timestamp
+
+
+def list_previously_deployed_shas(parsed_args, **kwargs):
+    service = parsed_args.service
+    deploy_groups = set([deploy_group for deploy_group in parsed_args.deploy_groups.split(',') if deploy_group])
```

Comment:
how does this actually work, do we have `deploy_groups` to work with at this point?